Makefile

```diff
@@ -72,13 +77,72 @@ test: generate/zz_filesystem_generated.go ## Run core unit tests
 	go test -race -cover -coverprofile=coverage.txt ./...
 
 .PHONY: check
-check: $(BIN_GOLANGCI_LINT) ## Check code quality (lint)
+check: check-lint check-goimports check-misspell check-whitespace check-eof ## Check code quality (comprehensive)
```

Comment:
Adds checkers to be as close to those from knative automations such that running `make check` locally is a reasonable approximation.